en/docs/learn/api-security/oauth2/token-persistence.md

```diff
@@ -107,11 +105,10 @@ The flow of the synchronous token persistence when receiving two identical acces
 4.  Alternatively, if an existing access token is not found, the OAuth2 component in **both nodes** creates a new access token and persists the access token to the database using the same thread.
 5.  One of the nodes will persist the token successfully and return it to the client but the other node will receive an error due to the violation of the CON_APP_KEY constraint.
 
-#### Recovery flow
+##### Recovery flow
 
 The process flow now moves on to the recovery flow described above in order to handle the `CON_APP_KEY` constraint violation and is executed as follows:
```

Comment:
Should this be "above" or "below"?

Reply:
I believe it is refers to the recovery flow of the CON_APP_KEY violation described in the above step. Hence, shall we change it as below:
```
 The process flow now moves on to the recovery flow in order to handle the `CON_APP_KEY` constraint violation and it is executed as follows: 
```
WDYT?

Reply:
+1